File: apps/android/app/src/main/java/com/cortexlumora/lunastories/LegalLinks.kt
package com.cortexlumora.lunastories

/**
 * Single source of truth for Terms & Privacy URLs on Android. Mirrors
 * iOS `apps/ios/Luna Stories/LegalLinks.swift` — keep these in sync
 * across both platforms so no app version ever links to a stale or
 * mismatched URL.
 */
object LegalLinks {
    const val TERMS_URL = "https://lunastories.cortexlumora.com/terms"
    const val PRIVACY_URL = "https://lunastories.cortexlumora.com/privacy"
}

File: apps/rn/src/lib/legal-links.ts
import * as WebBrowser from 'expo-web-browser';

// LegalLinks.swift ported.
export const LegalLinks = {
  termsURL: 'https://lunastories.cortexlumora.com/terms',
  privacyURL: 'https://lunastories.cortexlumora.com/privacy',
} as const;

export const legalLinks = {
  terms: LegalLinks.termsURL,
  privacy: LegalLinks.privacyURL,
} as const;

export function openLegalLink(url: string): void {
  WebBrowser.openBrowserAsync(url).catch(() => {});
}
